config/locales/notices/en.yml

```diff
@@ -1,29 +1,28 @@
 en:
   notices:
-    password_changed: Your password was successfully changed.
-    totp_configured: Authenticator app successfully configured
-    totp_disabled: Authenticator app disabled
+    password_changed: You changed your password
+    totp_configured: You have enabled an authentication app
+    totp_disabled: You disabled your authentication app
     log_in_consent:
       text: By signing in, you agree to %{app}’s %{link}
       link: Security Consent & Privacy Act Statement
-    phone_confirmation_successful: Congratulations! Your account is now more secure.
+    phone_confirmation_successful: You confirmed your phone number. Now your account is more secure!
```

Comment:
I'd like this message to feel like a high-five, because we are requiring users to put in some effort and I'd like them to feel good about it. Not sure if this is the right wording, though.